.readthedocs.yaml

```diff
@@ -0,0 +1,28 @@
+# .readthedocs.yaml
```

Comment:
Is there a way to keep this file not in the root directory but inside `templates/DO-178C`. Otherwise, we are hardcoding the whole repo to serve the needs of the `DO-178C` content. It might be okay for now but eventually there will be more standards, so we will hit the problems of ownership.

One way to overcome this would be to create a completely standalone repository dedicated to `DO-178C`. Then we would not have any conflicts with other standards. Thinking about this now, I kind of like this idea. What do you think?